### PR TITLE
Subtypes of `AggregateChecker` should override `getSupportedChecker`

### PR DESCRIPTION
**Pull request by jyoo980**

### CODE REVIEW

File: docs/manual/creating-a-checker.tex
Comment (jyoo980) [inline docs/manual/creating-a-checker.tex:1082]: I think this might have been a copy-and-paste error. There are no instances in the CF where an aggregate checker overrides `getImmediateSubcheckerClasses`

**Review by mernst — APPROVED** [review]
LGTM.  Thanks!
I would like to hold off merging until Suzanne approves the PR.

**Comment by jyoo980** [comment]
@mernst Sounds good to me, thanks!

**Review by smillst — APPROVED** [review]

**Pull request merged**
